bin/gwc/gwc.ml

```diff
@@ -180,27 +199,24 @@ let main () =
       else raise (Arg.Bad ("Don't know what to do with \"" ^ x ^ "\"")))
     (List.rev !files);
   if not !just_comp then (
-    let bdir =
-      if Filename.check_suffix !out_file ".gwb" then !out_file
-      else !out_file ^ ".gwb"
-    in
-    if (not !force) && Sys.file_exists bdir then (
-      Printf.eprintf
-        "The database \"%s\" already exists. Use option -f to overwrite it."
-        !out_file;
-      flush stderr;
-      exit 2);
-    Lock.control (Mutil.lock_file !out_file)
-      false ~onerror:Lock.print_error_and_exit (fun () ->
-        let bdir =
-          if Filename.check_suffix !out_file ".gwb" then !out_file
-          else !out_file ^ ".gwb"
-        in
+    let bdir = !Geneweb.GWPARAM.bpath bname in
+    Geneweb.GWPARAM.test_base bname;
+    Geneweb.GWPARAM.init_etc bname;
+    Lock.control (Mutil.lock_file bdir) false ~onerror:Lock.print_error_and_exit
+      (fun () ->
         let next_family_fun = next_family_fun_templ (List.rev !gwo) in
-        if Db1link.link next_family_fun bdir then ()
+        if Db1link.link next_family_fun bdir then (
+          Geneweb.Util.print_default_gwf_file bname;
```

Comment:
I suggest to set an empty gwf for gwc to avoid to change default parameters setting versus previous geneweb versions.